composer.json

```diff
@@ -27,7 +27,8 @@
 		"yoast/phpunit-polyfills": "^1.0"
 	},
 	"replace": {
-		"symfony/polyfill-mbstring": "*"
+		"symfony/polyfill-mbstring": "*",
+		"symfony/polyfill-intl-normalizer": "*"
```

Comment:
I don't believe this is necessary. The line was added for `polyfill-mbstring` since it's a [direct requirement](https://github.com/symfony/validator/blob/5.2/composer.json#L22) in the `symfony/validator` package. Either way it shouldn't cause any harm and could be an extra protection so it isn't loaded twice.